proto/akash/audit/v1beta2/query.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
```

Comment:
We can use a mix of versions - not everything necessarily needs to move to `v1beta2`.